ajax/libs/dagre/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "dagre",
+  "version": "0.7.4",
+  "filename": "dagre.min.js",
+  "description": "Graph layout for JavaScript",
+  "homepage": "https://github.com/cpettitt/dagre/wiki/",
```

Comment:
@pvnr0082t This is the link to the wiki including examples and configuration options which isn't included in the git repo main page. Seems better for user to keep it, no?